user.js

```diff
@@ -32,6 +32,13 @@ user_pref("dom.webaudio.enabled",				false);
 // https://www.mozilla.org/en-US/firefox/geolocation/
 user_pref("geo.enabled",					false);
 
+// PREF: When geolocation is enabled, use Mozilla geolocation service instead of Google
+// https://bugzilla.mozilla.org/show_bug.cgi?id=689252
+user_pref("geo.wifi.uri", "https://location.services.mozilla.com/v1/geolocate?key=test");
```

Comment:
On my Windows & FF53 the default is "https://www.googleapis.com/geolocation/v1/geolocate?key=%GOOGLE_API_KEY%"